docs/releasing_openttd.md

```diff
@@ -29,20 +29,25 @@ This guide is for OpenTTD developers/maintainers, to release a new version of Op
 
 1. Go to https://github.com/OpenTTD/website/new/main/_posts and write a new announcement post. See a [previous example](https://github.com/OpenTTD/website/pull/238) for a template.
 2. Create a new branch for this post and open a PR for it.
-3. Write announcement text for socials like Forum/Discord/Twitter/Reddit and include it in the PR.
+3. Write announcement text for socials like tt-forums / Discord / Twitter / Reddit / Fosstodon / ..., and include it in the PR.
 4. Create a Steam news image for that post and include it in the PR.
-5. Check the website post (preview link via checks page) and make corrections. We usually just use the GitHub web interface for this and squash the result later.
+5. Check the website post ("View Deployment" link) and make corrections. We usually just use the GitHub web interface for this and squash the result later.
 6. Get this PR approved, but do not merge yet.
 
 ## Step 3: Make the actual OpenTTD release
 
-1. Go to https://github.com/OpenTTD/OpenTTD/releases/new and create a new tag matching the release number. For the body of the release, see any older release. "Set as a pre-release" for a beta or RC, set as latest for a real release.
-2. Merge website PR.
-3. Wait for the OpenTTD release checks to be complete.
-4. Check that website links to the new release are working and correct, using the [staging website](https://www-staging.openttd.org/).
-5. If this is a full release, ask orudge to update the Microsoft Store and TrueBrain to move the release from the "testing" to "default" branch on Steam.
+1. Go to https://github.com/OpenTTD/OpenTTD/releases/new and create a new tag matching the release number. For the body of the release, copy in the changelog. "Set as a pre-release" for a beta or RC.
+2. Wait for the OpenTTD release workflow to be complete.
+3. Merge website PR.
```

Comment:
Is it worth noting here that this will make the post live immediately? (And is that even true? I got so confused by my recent forays into accidentally publishing posts with the wrong date. 😛)

Reply:
Sure.

And yes. You just published a post in the future, instead of the same day. For that you only got yourself to blame :)